index.html

```diff
@@ -37,13 +40,150 @@ <h1 class="supporting-text">I'm a software developer</h1>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
 
+    </section>
 
-    </ul>
+  </section>
+  <section class="about-me-section">
+    <div class="about-me">
+      <h3 class="about-title">About me</h3>
+      <p class="description-about">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </div>
+    <hr class="solidhr1">
+    <div class="skills">
+      <div class="about-me-sectiones">
```

Comment:
- [ ] Your design is impressive @sanieni6, But there is a minor detail missing for this section, As you can see from the screenshot your design could use more spacing between contents and your decoration image can pop-out to the view a bit.
### Your design 
![Screenshot 2023-01-14 081735](https://user-images.githubusercontent.com/69077061/212457304-90ed4af9-ab34-4374-a98c-bc055e1a064a.png)

### Figma design
![Screenshot 2023-01-14 081832](https://user-images.githubusercontent.com/69077061/212457306-a230ce5a-f949-4356-9cc0-15b7d68dd5a5.png)